### ISSUE TITLE
Lack of license

### ISSUE THREAD

**Issue by suy** [Beamdog/bgfileformats#1]
I suppose that if you have published this, you will not bring me to court if I'm trying to implement a library to support the Infinity Engine file formats, and I license it under some OSI/FSF-approved license even though I looked at this source code.

But the thing is that I am not a lawyer, so I would prefer if you had some usual open source license in this repo, so I would know if it's something I can look at to replicate, somewhat, in my own code with another license.

TL;DR: please, add a license. :-)
